misc/python/materialize/cloudtest/app/materialize_application.py

```diff
@@ -161,3 +161,70 @@ def set_environmentd_failpoints(self, failpoints: str) -> None:
         stateful_set.env["FAILPOINTS"] = failpoints
         stateful_set.replace()
         self.wait_for_sql()
+
+    def get_k8s_value(
+        self, selector: str, json_path: str, remove_quotes: bool = True
```

Comment:
`remove_quotes` is never set to False, so I suggest you remove it altogether. Instead, consider parsing the output as JSON and returning the parsed representation upstream.

Reply:
Not sure, still sounds a bit dangerous as default. Wrong if not just a plain value (leaf value) is returned. Need to think about it.